tests/system/small/test_null_index.py

```diff
@@ -0,0 +1,288 @@
+# Copyright 2024 Google LLC
```

Comment:
Moved from tests/system/small/test_empty_index.py and added `test_null_index_merge_*` tests.